interface/forms/vitals/templates/vitals/general_new.html

```diff
@@ -121,213 +134,239 @@
-		<td class='historicalvalues'>{if $result.waist_circ != 0}{$result.waist_circ}{/if}</td>
-	{/foreach}</tr>
-
-	{if $units_of_measurement == 3 || $gbl_vitals_options > 0}<tr class="hide">{else}<tr>{/if}
-                {if $units_of_measurement == 1}<td class="unfocus graph" id="waist_circ_metric">{else}<td class="graph" id="waist_circ_metric">{/if}{xl t="Waist Circumference"}</td>
-		{if $units_of_measurement == 1}<td class="unfocus">{else}<td>{/if}{xl t="cm"}</td>
-                {if $units_of_measurement == 1}<td class="valuesunfocus">{else}<td class='currentvalues'>{/if}
-                <input type="text" size='5' id='waist_circ_input_metric' value="{if $vitals->get_waist_circ() != 0}{math equation="round(number * constant,1)" number=$vitals->get_waist_circ() constant=2.54 format="%.2f"}{/if}" onChange="convCmtoIn('waist_circ_input');"/>
-                </td>
-        {foreach item=result from=$results}
-                <td class='historicalvalues'>{if $result.waist_circ != 0}{math equation="number * constant" number=$result.waist_circ constant=2.54 format="%.2f"}{/if}</td>
-        {/foreach}</tr>
-
-	<tr><td class="graph" id="BMI">{xl t="BMI"}</td><td>{xl t="kg/m^2"}</td>
-		<td class='currentvalues'><input type="text" size='5'
-			name='BMI' id='BMI_input' value="{if $vitals->get_BMI() != 0}{$vitals->get_BMI()|substr:0:5}{/if}"/></td>
-	{foreach item=result from=$results}
-		<td class='historicalvalues'>{if $result.BMI != 0}{$result.BMI|substr:0:5}{/if}</td>
-	{/foreach}</tr>
-
-  <tr><td>{xl t="BMI Status"}</td><td>{xl t="Type"}</td>
-    <td class='currentvalues'><input type="text" size='15'
-    name="BMI_status" id="BMI_status' value="{$vitals->get_BMI_status()}"/></td>
-  {foreach item=result from=$results}
-    <td  class='historicalvalues'>{if $result.BMI_status}{xl t=$result.BMI_status}{/if}</td>
-  {/foreach}
-  </tr>
-
-  <tr><td>{xl t="Other Notes"}<td></td></td>
-    <td class='currentvalues'><input type="text" size='20'
-      name="note" id='note' value="{$vitals->get_note()}" /></td>
-  {foreach item=result from=$results}
-    <td class='historicalvalues'>{$result.note}</td>
-  {/foreach}</tr>
-
-	<tr>
-	<td colspan='3' style='text-align:center'>
-	{if $patient_age <= 20 || (preg_match('/month/', $patient_age))}
-	<!-- only show growth-chart button for patients < 20 years old -->
-	<!-- <input type="button" id="growthchart" value="{xl t="Growth-Chart"}" style='margin-left: 20px;'> -->
-	<input type="button" id="pdfchart" value="{xl t="Growth-Chart"} ({xl t="PDF"})" style='margin-left: 20px;'>
-        <input type="button" id="htmlchart" value="{xl t="Growth-Chart"} ({xl t="HTML"})" style='margin-left: 20px;'>
-	{/if}
-	</td>
-	</tr>
-
-	<tr><td colspan='3' style='text-align:center'>&nbsp;</td></tr>
-	<tr>
-	<td colspan='3' style='text-align:center'>
-        <input type="submit" class="editonly" name="Submit" value="{xl t="Save"}">
-        <input type="button" class="editonly" id="cancel" value="{xl t="Cancel"}">
```

Comment:
Note this editonly class is used to hide things that should not be in the trend/graph vitals page (this is the Trend button on the patient summary screen)

Reply:
Added